packages/neos-ui-ckeditor5-bindings/src/ckEditorApi.js

```diff
@@ -78,7 +78,13 @@ export const createEditor = store => options => {
 
             editor.model.document.on('change', () => handleUserInteractionCallback());
             editor.model.document.on('change:data', debounce(() => onChange(cleanupContentBeforeCommit(editor.getData())), 500, {maxWait: 5000}));
-        }).catch(e => console.error(e));
+        }).catch(e => {
+            if (e instanceof TypeError && e.message.match(/Class constructor .* cannot be invoked without 'new'/)) {
+                console.error("Neos.Ui: Youre probably using a CKeditor plugin which needs to be rebuild.\nsee https://github.com/neos/neos-ui/issues/3287\n\nOriginal Error:\n\n" + e.stack);
```

Comment:
`\nsee` is this a typo? shoudlnt it be `\n\see` ?

Reply:
Hi ;) no I believe it’s called something like c-style character in strings escape (works in double quotes) and \n is equivalent to ascii 10 or simply put a newline ;)